### DISCUSSION TITLE
Airflow DAG run missing on few days

### DISCUSSION THREAD

**Discussion by bmenugul** [apache/airflow#41978] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### If "Other Airflow 2 version" selected, which one?

2.6.3

### What happened?

Schedule DAG execution missing in Airflow. Next Run of the DAG is skipped. In the Airflow UI DAG calendar view has 'no_status'

### What you think should happen instead?

As per the schedule, the DAG should have run on the date but dag run is skipped.

### How to reproduce

Currently this issue is found on multiple DAGs but currently looks like it is difficult to reproduce.

### Operating System

GCP composer

### Versions of Apache Airflow Providers

Version 2.6.3

### Deployment

Google Cloud Composer

### Deployment details

Airflow Version: 2.6.3
Composer version: 2.5.4

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by nathadfield** [comment]
Without a reproducible example or, at the very least more information and context, I doubt you are going to get much help on this one because this is not a known problem or anything that is commonly experienced by Airflow users which hints that it might be something that is specific to you.

**Comment by potiuk** [comment]
Yep. Converted to discussion - because without reproducible details, it's at most discussion.
